apps/meteor/client/views/marketplace/AppDetailsPage/AppDetailsPageTabs.tsx

```diff
@@ -11,15 +11,23 @@ type AppDetailsPageTabsProps = {
 	isSecurityVisible: boolean;
 	settings: ISettings | undefined;
 	tab: string | undefined;
+	hasCluster: boolean;
 };
 
-const AppDetailsPageTabs = ({ context, installed, isSecurityVisible, settings, tab }: AppDetailsPageTabsProps): ReactElement => {
+const AppDetailsPageTabs = ({
+	context,
+	installed,
```

Comment:
Why not?

```suggestion
	installed = false,
```

It avoids the confusing `Boolean(installed)` calls.